SECURITY.md

```diff
@@ -0,0 +1,46 @@
+# Security policy
+
+## Supported versions
+<!-- Include start supported versions -->
+The release model of the core22 snap is following rolling releases. 
+The snap is released into the edge channel, and automatically promoted to 
+the beta channel if a snap revision is not already in progress for beta validation. 
+When reporting security issues against the core22 snap, only the latest 
+release of the core22 snap is supported.
+
+The core22 snap has regular releases that are fully automated. There are two 
+types of security fixes that can be shipped with new versions of the core22 snap.
+
+- Security fixes that are relevant to the files inside this repository.
+- Security fixes that are carried from the official archives. I.e security fixes 
+from debian packages carried inside the core22 snap.
+
+<!-- Include end supported versions -->
+
+## What qualifies as a security issue
+
+Security vulnerability that apply to packages in the Jammy archives also shipped by the
+core22 snap. Any vulnerability that allows the core22 snap to interfere outside 
+of the intended restrictions also qualifies as a security issue.
+
+## Reporting a vulnerability
+
+The easiest way to report a security issue is through
+[GitHub](https://github.com/canonical/core-base/security/advisories/new). See
+[Privately reporting a security
+vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+for instructions.
+
```

Comment:
I suggest removing the security@ubuntu.com part and sticking with github security advisories to coordinate communications.